internal/controller/nnf_node_block_storage_controller.go

```diff
@@ -587,14 +629,57 @@ func (r *NnfNodeBlockStorageReconciler) deleteStorageGroup(ss nnf.StorageService
 	return ss.StorageServiceIdStorageGroupIdDelete(ss.Id(), id)
 }
 
+// Enqueue all the NnfNodeBlockStorage resources after an nnf-ec node-up/node-down event. If we
+// can't List() the NnfNodeBlockStorages, trigger the watch again after 10 seconds.
+func (r *NnfNodeBlockStorageReconciler) NnfEcEventEnqueueHandler(ctx context.Context, o client.Object) []reconcile.Request {
+	log := r.Log.WithValues("Event", "Enqueue")
+
+	requests := []reconcile.Request{}
+
+	// Find all the NnfNodeBlockStorage resources for this Rabbit so we can reconcile them.
+	listOptions := []client.ListOption{
+		client.InNamespace(os.Getenv("NNF_NODE_NAME")),
+	}
+
+	nnfNodeBlockStorageList := &nnfv1alpha2.NnfNodeBlockStorageList{}
+	if err := r.List(context.TODO(), nnfNodeBlockStorageList, listOptions...); err != nil {
+		log.Error(err, "Could not list block storages")
```

Comment:
How common will this be? Do we really want the backtrace?

Reply:
These events aren't that common, and a List() failing isn't very common, but if you're worried I can change it to an Info() with the error message included.